source/main.cpp

```diff
@@ -27,6 +27,9 @@ extern "C" void _initterm(_PVFV *beg, _PVFV *end);
 
 HANDLE console = INVALID_HANDLE_VALUE;
 char blink_pipe_name[MAX_PATH];
+#define MAX_ENVIRONMENT_LENGTH 32767 // max for Windows
+TCHAR blink_environment[MAX_ENVIRONMENT_LENGTH];
+TCHAR blink_working_directory[MAX_PATH];
```

Comment:
These (+ the `run` arguments) should be `WCHAR`, since they are always filled with unicode strings (and `CreateProcess` is later executed with `CREATE_UNICODE_ENVIRONMENT`) and therefore must not contain ANSI strings.

Reply:
Thanks. Fixed.